### DISCUSSION TITLE
Allow (publicly) sharing collection page

### DISCUSSION THREAD

**Discussion by alrick** [directus/directus#15253] [Feature Requests]
Is it possible to create unauthenticated views inside the app ?

It would be awesome to be able to define an "unauthenticated" role like the public role and setup the permissions accordingly.
That way, we could have a public access mainly for viewing purpose.
I don't think it's possible atm but maybe with extensions ?

**Comment by rijkvanzanten** [comment]
We already support this type of public sharing of data from the item detail page 🙂 See https://docs.directus.io/app/content/shares.html#shares for examples and more information

**Reply by alrick** [reply]
What I mean is collection public views, not just only item details.

We use directus to build databases at the University of Fribourg and a lot of data is open-access. It would be really awesome to let users access all the available data without auth. We could define a specific role and chose which data is public or not same as permission for other roles and maybe have a specific url to consult open-access collections within a database. That way unauthenticated users could use all the feature of the collection listing (filters, listings, search, etc.)

**Reply by rijkvanzanten** [reply]
Ahh gotcha! Yeah that'd be a great enhancement 🙂  Lemme recategorize this Q&A to be the feature request for that 👍🏻

**Comment by rijkvanzanten** [comment]
Somewhat related to https://github.com/directus/directus/discussions/14949

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
